components/map/MapAutocomplete.tsx

```diff
@@ -1,29 +1,60 @@
-import { useCallback } from 'react'
+import { BottomSheetFlatList } from '@gorhom/bottom-sheet'
+import { forwardRef, useCallback } from 'react'
+import { ListRenderItem, TextInput as RNTextInput, View } from 'react-native'
 
-import Autocomplete from '@/components/inputs/Autocomplete'
-import { MapRef } from '@/components/map/Map'
+import Autocomplete, { AutocompleteProps } from '@/components/inputs/Autocomplete'
+import FlexRow from '@/components/shared/FlexRow'
+import Icon from '@/components/shared/Icon'
+import Typography from '@/components/shared/Typography'
+import { useTranslation } from '@/hooks/useTranslation'
 import { GeocodingFeature } from '@/modules/map/types'
 import { forwardGeocode } from '@/modules/map/utils/forwardGeocode'
 
-type Props = {
-  setFlyToCenter?: MapRef['setFlyToCenter']
-}
+type Props = Partial<AutocompleteProps<GeocodingFeature>>
 
-const MapAutocomplete = ({ setFlyToCenter }: Props) => {
-  const handleValueChange = useCallback(
-    (value: GeocodingFeature) => {
-      setFlyToCenter?.(value.center)
-    },
-    [setFlyToCenter],
-  )
+const MapAutocomplete = forwardRef<RNTextInput, Props>(
+  ({ onValueChange, ...restProps }: Props, ref) => {
+    const t = useTranslation('ZoneDetailsScreen')
+    const handleValueChange = useCallback(
+      (value: GeocodingFeature) => {
+        onValueChange?.(value)
+      },
+      [onValueChange],
+    )
 
-  return (
-    <Autocomplete
-      getOptions={forwardGeocode}
-      getOptionLabel={(option) => option.place_name || option.text}
-      onValueChange={handleValueChange}
-    />
-  )
-}
+    const renderItem: ListRenderItem<GeocodingFeature> = useCallback(
+      ({ item }) => (
+        <View className="border-b-[2px] border-divider py-4">
```

Comment:
2px is default :) 
![image](https://github.com/bratislava/paas-mpa/assets/19418224/e50ff656-d913-4608-9e6b-853ecb3de895)